<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<bug>
    <id>1220</id>
    <title>Error pulling from a private docker registry server</title>
    <description>
        <paragraph id="1">
            <sentence id="1.1">After pushing to a private registry, I attempt to pull the same image from a different host and receive the following error on the client:</sentence>
        </paragraph>
        <paragraph id="3">
            <sentence id="3.1">Here's a dump of the request/response that seems to be failing:</sentence>
        </paragraph>
    </description>
</bug>
